tools/scripts/uv-export.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -o errexit
+
+dnf install -y python3.12-devel gcc git
+
+# Install uv
+curl -LsSf https://astral.sh/uv/install.sh | sh
+source $HOME/.local/bin/env
+
+# Generate uv.lock from pyproject.toml (respects [tool.uv] settings)
+uv lock --python 3.12
```

Comment:
I'm not sure if I fully understand your comment. Do we actually need to pass the `--python 3.12`, since the Python version is defined in `pyproject.toml`?